scripts/install

```diff
@@ -167,20 +163,18 @@ installUrl="/admin/install/index.php"
 
 ynh_local_curl $installUrl "u_email=$email" "u_firstname=$firstname" "u_name=$lastname" "u_login=$admin" "u_pwd=$password" "u_pwd2=$password"
 
-ynh_app_setting_delete $app skipped_uris
-
 #=================================================
 # SETUP SSOWAT
 #=================================================
 
-# Make app public if necessary
-if [ $is_public -eq 1 ]; then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set $app unprotected_uris "/"
-	# protect admin area
-	ynh_app_setting_set $app protected_uris "/admin"
+if [ $is_public -eq 0 ]; then
+	ynh_permission_update --permission "main" --remove "visitors" --add "all_users"
 fi
 
+ynh_permission_create --permission "admin"
```

Comment:
@alexAubin If I add --url "/admin" here I got an error 504 when I try to go to the admin panel. What I'm doing wrong here?

Reply:
```
# Delete skipped_uris if it exists
if [ ! -z $skipped_uris ]; then
	ynh_app_setting_delete $app skipped_uris
fi

# Delete unprotected_uris if it exists
if [ ! -z $unprotected_uris ]; then
	ynh_app_setting_delete $app unprotected_uris
fi

# Delete protected_uris if it exists
if [ ! -z $protected_uris ]; then
	ynh_app_setting_delete $app protected_uris
fi
```

Fix the issue, sorry for the noise